python/ray/autoscaler/_private/event_system.py

```diff
@@ -63,11 +66,16 @@ def add_callback_handler(
                 registered against.
             callback (Callable[[Dict], None]): Callable object that is invoked
                 when specified event occurs.
+            *args: Variable length arguments to be injected into callbacks.
+            **kwargs: Keyword arguments to be injected into callbacks.
         """
         if event not in CreateClusterEvent.__members__.values():
             cli_logger.warning(f"{event} is not currently tracked, and this"
                                " callback will not be invoked.")
 
+        if args is not None or kwargs is not None:
```

Comment:
I think args is always an empty list if not specified, and kwargs is always an empty dict, so I think we can drop the if here and have the same result for all cases.